app/views/forms/delete_confirmation/delete.html.erb

```diff
@@ -5,7 +5,8 @@
     <%= f.govuk_error_summary %>
   <% end %>
   <span class="govuk-caption-l"><%= @item_name %></span>
-  <%= f.govuk_collection_radio_buttons :confirm_deletion, @confirm_deletion_options, :value,
+  <%= f.govuk_collection_radio_buttons :confirm,
```

Comment:
I'm not clear what's going on here?

Reply:
I wasn't either, and I'm still not entirely, but it's following this https://govuk-form-builder.netlify.app/form-elements/radios/#populating-values-labels-and-hints-with-procs

Reply:
ah ok... thanks for pointing me to that! had no idea.